src/modules/deposit/service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from "@nestjs/common";
+import { InjectRepository } from "@nestjs/typeorm";
+import { Repository } from "typeorm";
+import { Deposit } from "../scraper/model/deposit.entity";
+
+@Injectable()
+export class DepositService {
+  constructor(@InjectRepository(Deposit) private depositRepository: Repository<Deposit>) {}
+
+  public async getDeposits(status = null, limit = 10, offset = 0) {
+    const where = status ? { status } : undefined;
```

Comment:
let's check for the depositDate column to not be `null`

Reply:
done